web3swift/src/Account/EthereumAccount.swift

```diff
@@ -76,6 +76,19 @@ public class EthereumAccount: EthereumAccountProtocol {
             throw EthereumAccountError.createAccountError
         }
     }
+
+	public static func importAccount(keyStorage: EthereumKeyStorageProtocol, privateKey: String, keystorePassword password: String) throws -> EthereumAccount {
```

Comment:
Can you change `privateKey` to be `Data`, then you'll avoid the throw below.

Reply:
You are right!

Reply:
Sure, can be either way